src/components/subscriptions/licenses/LicenseAllocationHeader.jsx

```diff
@@ -17,14 +18,28 @@ const LicenseAllocationHeader = () => {
   return (
     <>
       {isBrowseAndRequestFeatureAlertShown && <NewFeatureAlertBrowseAndRequest />}
-      <h2 className="mb-2">License Allocation</h2>
+      <h2 className="mb-2">{intl.formatMessage({
+        id: 'admin.portal.license.allocation.header',
+        defaultMessage: 'License Allocation',
+        description: 'Header for the license allocation section.',
+      })}
+      </h2>
       <p className="lead">
-        {subscription.licenses?.allocated}
-        {' of '}
-        {subscription.licenses?.total} licenses allocated
+        {intl.formatMessage({
+          id: 'admin.portal.license.allocation.allocated.licenses.count',
+          defaultMessage: '{allocated} of {total} licenses allocated',
+          description: 'Text for the number of licenses allocated.',
+        }, {
+          allocated: subscription.licenses?.allocated,
+          total: subscription.licenses?.total,
+        })}
       </p>
     </>
   );
 };
 
-export default LicenseAllocationHeader;
+LicenseAllocationHeader.propTypes = {
+  intl: intlShape.isRequired,
+};
+
+export default injectIntl(LicenseAllocationHeader);
```

Comment:
As this is a functional component so you don't need to inject Intl here. You can directly use FormattedMessage  here by importing.